hphp/runtime/vm/jit/vasm-simplify.cpp

```diff
@@ -196,6 +196,37 @@ bool simplify(Env& env, const setcc& vsetcc, Vlabel b, size_t i) {
   });
 }
 
+template <typename Reg>
+bool operand_one(Env& env, Reg op) {
```

Comment:
@swalk-cavium—What I mean is that you can do as @cmuellner suggested and implement `csinc*{}` instead of `csone*{}`—you can just specify the `vixl::wzr` operand explicitly.